ui/imports/shared/popups/addaccount/panels/WatchOnlyAddressSection.qml

```diff
@@ -24,10 +26,30 @@ Column {
         id: d
 
         property bool incorrectChecksum: false
+        readonly property string uuid: Utils.uuid()
+        property string resolvedEnsAddress
 
         function checkIfAddressChecksumIsValid(address) {
             d.incorrectChecksum = !root.store.isChecksumValidForAddress(address)
         }
+
+        readonly property var validateEnsAsync: Backpressure.debounce(root, 500, function (value) {
+            var name = value.startsWith("@") ? value.substring(1) : value
+            root.store.mainModuleInstance.resolveENS(name, d.uuid)
+        });
+    }
+
+    Connections {
```

Comment:
same here we can use the connect function to connect the signal from the store in component on completed